### ISSUE TITLE
abnormal aspartoacylase level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3585]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

A. abnormal aspartoacylase level
B. increased aspartoacylase level
C. decreased aspartoacylase level

>2. Synonyms (e.g., absent spleen)

A. aberrant aspartoacylase level; altered aspartoacylase level; abnormal N-acetylaspartate amidohydrolase level; abnormal N-acyl-L-aspartate amidohydrolase level; abnormal acetyl-aspartic deaminase level; abnormal acylase II level; abnormal aminoacylase II level
B. increased N-acetylaspartate amidohydrolase level; increased N-acyl-L-aspartate amidohydrolase level; increased acetyl-aspartic deaminase level; increased acylase II level; increased aminoacylase II level
C. reduced aspartoacylase level; decreased N-acetylaspartate amidohydrolase level; decreased N-acyl-L-aspartate amidohydrolase level; decreased acetyl-aspartic deaminase level; decreased acylase II level; decreased aminoacylase II level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

A. aberrant concentration of the enzyme that catalyzes the the reaction: N-acyl-L-aspartate + H2O = a fatty acid anion + L-aspartate.
B. increased concentration of the enzyme that catalyzes the the reaction: N-acyl-L-aspartate + H2O = a fatty acid anion + L-aspartate.
C. reduced concentration of the enzyme that catalyzes the the reaction: N-acyl-L-aspartate + H2O = a fatty acid anion + L-aspartate.

PMID:10820213

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

A. abnormal enzyme/coenzyme level (MP:0005319)
B. abnormal aspartoacylase level (new term)
C. abnormal aspartoacylase level (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-680

**Comment by slaulederkind** [comment]
ORCID:0000-0003-3645-6803

**Comment by anna-anagnostop** [comment]
Added new terms:
abnormal aspartoacylase level MP:0031396
increased aspartoacylase level MP:0031397
decreased aspartoacylase level MP:0031398
